src/configure_config.cmake

```diff
@@ -160,6 +160,9 @@ endif ()
 if (ENABLE_OPENSSL)
     set(USE_OPENSSL_INTREE 1)
 endif ()
+if (TARGET ch_contrib::stl-cpp)
```

Comment:
"std-cpp" is too easy to confuse with C++'s STL. Maybe call this "ch_contrib::seasonal_decompose" or something similarly verbose